src/ui/analytics/chart/filterNameOption.tsx

```diff
@@ -0,0 +1,28 @@
+import { useState } from "react";
```

Comment:
this component doesn't have the correct name, it should be named ........   .ui.tsx